.cursor/rules/javascript-development.mdc

```diff
@@ -0,0 +1,42 @@
+---
+description: JavaScript/TypeScript development patterns for sentry-cli npm wrapper
+globs: js/**/*,scripts/**/*,package.json,*.js
+alwaysApply: false
+---
+
+# JavaScript Development Guidelines
+
+## Code Organization
+
+- Maintain compatibility with existing npm package structure
+- Update TypeScript definitions in `js/index.d.ts` if needed
+- Consider impact on installation flow in `scripts/install.js`
+- Test across different Node.js versions
+
+## Installation & Distribution
+
+- Multiple distribution methods: npm, direct binary, package managers
```

Comment:
l: not sure what this rule is about regarding JS, so maybe something to get rid of? (happy to let you make the call)

```suggestion
```